include/ck/tensor_operation/gpu/device/device_reduce_multiblock.hpp

```diff
@@ -34,7 +34,8 @@ template <typename InDataType,
           index_t KThreadSliceSize,
           index_t InSrcVectorDim,
           index_t InSrcVectorSize,
-          index_t OutDstVectorSize>
+          index_t OutDstVectorSize,
+          bool MultiBlockReduction = false>
```

Comment:
Actually it can be removed. Fixed in https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/284/commits/1786b830abc882fbd994958adcc216ed01c5f858